src/ducks/wallet/trezor.ts

```diff
@@ -0,0 +1,96 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+// @types/trezor-connect doesn't have .stellarGetAddress()
+// @ts-ignore
+import TrezorConnect from "trezor-connect";
+
+export const fetchTrezorStellarAddressAction = createAsyncThunk<
+  // Return type of the payload creator
+  string,
+  // First argument to the payload creator
+  undefined,
+  // Types for ThunkAPI
+  { rejectValue: RejectMessage }
+>(
+  "wallet/trezor/fetchTrezorStellarAddressAction",
+  async (_, { rejectWithValue }) => {
```

Comment:
`_` with type `undefined` is there to handle "thunk arguments" which we don't need in this case. I don't think there is a way to not include it there if we need to use "thunk API" (rejectValue).